### DISCUSSION TITLE
Relationship / Comparison to Pimcore Datahub CI Hub integration

### DISCUSSION THREAD

**Discussion by cancan101** [BrandOriented/PimcoreCiHubConnector#16] [General]
What is the relationship / comparison to the [Pimcore Enterprise feature "Datahub CI Hub integration"](https://pimcore.com/docs/platform/Datahub_CI_Hub/)?

It looks like there is also an [Pimcore Enterprise feature "Datahub Simple Rest API"](https://pimcore.com/docs/platform/Datahub_Simple_Rest/).

**Comment by labudzinski** [comment]
In the current version of the base on https://github.com/CI-HUB-GmbH/PimcoreCIHubAdapterBundle - while it is not used only to display assets and objects - more description on: https://ci-hub.com/blog/the-all-new-pimcore-integration-releases-premium-features-to-all

**Reply by cancan101** [reply]
Thank you for the pointer. And how does your integration compare to the Bundle that Pimcore offers as part of their Enterprise package?

**Reply by cancan101** [reply]
Also is there any compatibility or incompatibility with Pimcore's Pimcore Datahub Simple Rest API: https://docs.pimcore.com/platform/Datahub_Simple_Rest/ ?
